--- origsrc/libgadu-1.11.0/test/automatic/Makefile.am	2011-05-29 08:34:00.000000000 -0500
+++ src/libgadu-1.11.0/test/automatic/Makefile.am	2011-07-01 19:04:10.090431100 -0500
@@ -27,8 +27,8 @@ EXTRA_DIST = protocol.txt connect.txt
 
 SUBDIRS = script
 
-script/script.c:       $(wildcard script/*.scr) script/compile
-	$(PERL) script/compile script/*.scr > script/script.c
+script/script.c:       $(wildcard $(srcdir)/script/*.scr) script/compile
+	$(PERL) $(srcdir)/script/compile $(srcdir)/script/*.scr > script/script.c
 
 clean-local:
 	rm -f *.html *.log *-valgrind
