--- origsrc/libgadu-1.12.0/configure.ac	2014-11-20 00:03:52.026825800 -0600
+++ src/libgadu-1.12.0/configure.ac	2014-11-20 00:04:56.186914300 -0600
@@ -43,7 +43,13 @@ dnl  Egzotyczne systemy operacyjne
 dnl
 is_mingw="no"
 case "$host" in
-	*-*-mingw* | *-*-cygwin*)
+	*-*-cygwin*)
+		MINGW_LDFLAGS="-no-undefined"
+		MINGW_LIBGEN=""
+		MINGW_LIBDATA=""
+		LDFLAGS_NO_INSTALL="-no-fast-install"
+		;;
+	*-*-mingw*)
 		MINGW_LDFLAGS="-no-undefined -lws2_32"
 		MINGW_LIBGEN="-Wl,--output-def,libgadu.def.in"
 		MINGW_LIBDATA="libgadu.def"
