--- origsrc/libgadu-1.8.2/m4/acx_pthread.m4	2008-05-31 15:16:54.000000000 -0500
+++ src/libgadu-1.8.2/m4/acx_pthread.m4	2009-05-03 22:45:52.609250000 -0500
@@ -210,7 +210,7 @@ if test "x$acx_pthread_ok" = xyes; then
 		# FIXME: -fPIC is required for -shared on many architectures,
 		# so we specify it here, but the right way would probably be to
 		# properly detect whether it is actually required.
-		CFLAGS="-shared -fPIC -Wl,-z,defs $CFLAGS $PTHREAD_CFLAGS"
+		CFLAGS="-shared $CFLAGS $PTHREAD_CFLAGS"
 		LIBS="$PTHREAD_LIBS $LIBS"
 		CC="$PTHREAD_CC"
 
