--- origsrc/libgadu-1.7.1/m4/acx_pthread.m4	2005-11-02 15:39:49.000000000 -0600
+++ src/libgadu-1.7.1/m4/acx_pthread.m4	2007-05-27 13:34:37.630014100 -0500
@@ -210,7 +210,7 @@
 		# FIXME: -fPIC is required for -shared on many architectures,
 		# so we specify it here, but the right way would probably be to
 		# properly detect whether it is actually required.
-		CFLAGS="-shared -fPIC -Wl,-z,defs $CFLAGS $PTHREAD_CFLAGS"
+		CFLAGS="-shared $CFLAGS $PTHREAD_CFLAGS"
 		LIBS="$PTHREAD_LIBS $LIBS"
 		CC="$PTHREAD_CC"
 
--- origsrc/libgadu-1.7.1/src/Makefile.am	2007-04-21 18:29:18.000000000 -0500
+++ src/libgadu-1.7.1/src/Makefile.am	2007-05-27 13:10:57.043609900 -0500
@@ -1,4 +1,4 @@
 lib_LTLIBRARIES = libgadu.la
 libgadu_la_SOURCES = common.c dcc.c events.c http.c obsolete.c pubdir.c pubdir50.c libgadu.c
 libgadu_la_CFLAGS = -I$(top_srcdir) -I$(top_srcdir)/include
-libgadu_la_LDFLAGS = -version-number 3:7
+libgadu_la_LDFLAGS = -version-number 3:7 -no-undefined
